scripts/install_cairo_bindings.sh

```diff
@@ -24,7 +23,6 @@ function install_dev() {
 }
 
 function install_prod() {
-  git pull --recurse-submodules
```

Comment:
can you explain this change and how it is related?

Reply:
[Revert unwanted change, delete intermediate function and rename deplo…](https://github.com/software-mansion/protostar/pull/1845/commits/61d9b414d84f482849a114f029114024adb8c1ee)